superglue/run_superglue.sh

```diff
@@ -14,9 +14,13 @@ LOGPATH=${3:-logs}
 SEED=${4:-111}
 GPU=${5:-0}
 
+# for pretraining
+MNLICKPT=${6}
+SWAGCKPT=${7}
```

Comment:
can we use one argument here since this script only support single task? e.g., CKPT=${6}

Does this script if I didn't specific anything for CKPT?

Reply:
Can you also update usage part of the script?